inappmessaging/USERGUIDE.md

```diff
@@ -218,6 +218,17 @@ InAppMessaging.instance().onVerifyContext = { contexts: List<String>, campaignTi
 }
 ```
 
+### <a name="close-campaign"></a> #2 Close campaign API
+There are cases that apps need to manually close the campaigns without user interaction.
+
+An example is when a different user logs-in and the currently displayed campaign does not target the new user.
+
+It is possible that the new user did not close the campaign (tapping the 'X' button) when logging in. The app can force-close the campaign by calling the `closeMessage()` API.
```

Comment:
Is it worth adding an info that closed campaign message will not be counted as displayed?

Reply:
Yes, it is better to add.
Added, please check.